src/test/java/com/microsoft/azure/servicebus/jms/aad/QueueTest.java

```diff
@@ -1,6 +1,6 @@
 package com.microsoft.azure.servicebus.jms.aad;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
also not for here but for description please add a line to say:

The qpid.jms.client has been updated to the highest version that still supports the javax.jms namespace